src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
Right but we sort of don't want to use Backbone at all in this project. It's unfortunate we need to for using `artsy-passport`.

Checking the user has submitted or is a confirmed buyer can be done directly with `request` like in this example, no Backbone needed.